test-system.sh

```
@@ -0,0 +1,123 @@
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Contadores
+TESTS_PASSED=0
+TESTS_FAILED=0
+
+# Função para testar
+test_component() {
+    local name="$1"
+    local command="$2"
+    local expected="$3"
```

Shellcheck (0.10.0)

[warning] expected appears unused. Verify use (or export if used externally).

(SC2034)